src/_gettsim_tests/test_data/kindergeld.csv

```diff
@@ -1,4 +1,4 @@
-hh_id,tu_id,p_id,alter,arbeitsstunden_w,kind,in_ausbildung,bruttolohn_m,_zu_verst_eink_ohne_kinderfreib_tu,jahr,kindergeld_m_tu,kinderbonus_m_tu,kindergeld_m_hh,kinderbonus_m_hh,kinderbonus_m_tu,note
+hh_id,tu_id,p_id,alter,arbeitsstunden_w,kind,in_ausbildung,bruttolohn_m,_zu_verst_eink_ohne_kinderfreib_tu,jahr,kindergeld_m_tu,kinderbonus_m_tu,kindergeld_m_hh,kinderbonus_m_hh,kinderbonus_m_tu2,note
```

Comment:
```suggestion
hh_id,tu_id,p_id,alter,arbeitsstunden_w,kind,in_ausbildung,bruttolohn_m,_zu_verst_eink_ohne_kinderfreib_tu,jahr,kindergeld_m_tu,kinderbonus_m_tu,kindergeld_m_hh,kinderbonus_m_hh,kinderbonus_m_tu,note
```

?

Or was that on purpose?

Reply:
No, that was not on purpose, that must have been changed automatically, why I don't know.. But I will correct it!

Reply:
Excellent, thank you!